odoo/addons/test_lint/tests/_odoo_checker_gettext.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import astroid
+from pylint import checkers, interfaces
+
+
+class OdooBaseChecker(checkers.BaseChecker):
+    __implements__ = interfaces.IAstroidChecker
+    name = 'odoo'
+
+    msgs = {
+        'E8502': (
+            'Bad usage of _, _lt function.',
+            'gettext-variable',
+            'See https://www.odoo.com/documentation/13.0/reference/translations.html#variables'
```

Comment:
by the way, should I put `14.0` in the URL? It currently makes a 404 but it will contains the new syntax there. There is already the `master` URL but it's changing.

Reply:
I would say yes, that makes sense.